src/components/PeopleAndRoles/PeopleAndRoles.vue

```diff
@@ -439,6 +439,14 @@ export default class PeopleAndRoles extends Vue {
   private onPeopleAndRoles (): void {
     this.setPeopleAndRolesValid(this.hasValidRoles)
   }
+
+  /**
+   * Sets the 'changes' flag when a user has made changes.
+   */
+  @Watch('hasChanges')
+  private onPeopleAndRolesChanges (): void {
+    this.setPeopleAndRolesChanged(this.hasChanges)
```

Comment:
On the one hand, this will store the changed flag so we know that we can enable some actions.

On the other hand, this (and other components like it) are not actually identifying that changes were made since the last save -- the idea being that we could prompt the user for unsaved changes. Now if they resume a draft with changes, they will get prompted. Solving this will be tricky. Maybe we can live with this for now.